docs/tutorials/PipelinesApi101.md

```diff
@@ -27,13 +27,14 @@ You may create a service account using the `gcloud` command, consider running th
 
 ```
 #!/bin/bash
+export LC_ALL=C 
 RANDOM_BUCKET_NAME=$(head /dev/urandom | tr -dc a-z | head -c 32 ; echo '')
 
-#Create a new service account called "MyServiceAccount", and from the output of the command, take the email address that was generated
-EMAIL=$(gcloud beta iam service-accounts create MyServiceAccount --description "to run cromwell"  --display-name "cromwell service account" --format json | jq '.email' | sed -e 's/\"//g')
+#Create a new service account called "my-service-account", and from the output of the command, take the email address that was generated
+EMAIL=$(gcloud beta iam service-accounts create my-service-account --description "to run cromwell"  --display-name "cromwell service account" --format json | jq '.email' | sed -e 's/\"//g')
 
 # add all the roles to the service account
-for i in storage.objectCreator storage.objectViewer genomics.pipelinesRunner genomics.admin iam.serviceAccountUser storage.objects.create
+for i in storage.objectCreator storage.objectViewer lifesciences.workflowsRunner lifesciences.admin iam.serviceAccountUser storage.objects.create
```

Comment:
According to documentation, these are the replacements. Note that I didn't actually use these steps to create my account, as my dev account was already linked to a project with appropriate permissions. Therefore I'm just hoping that these changes are sufficient.

https://cloud.google.com/life-sciences/docs/how-tos/migration